packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -120,7 +128,7 @@ export const mockQueryEndpoint = (
     statusCode = 200
 ) => {
     const matcher = (uri: string) => uri.includes(matchingPath)
-    return nock(DEFAULT_TEST_HOST).get(matcher).reply(statusCode, response)
+    return nock(DEFAULT_TEST_HOST).get(matcher).delay(50).reply(statusCode, response)
```

Comment:
Minor: Since the delay is used in more than 1 function, how about extracting the delay amount into its own variable?